docs/how-to-test-parcels-and-preview-scenes.md

```diff
@@ -20,28 +20,57 @@ Note that the test-scenes building may take a while (10 mins or more).
 
 4. To spawn at the specific coordinates of a test scene append the position query paramter:
 
-    http://localhost:3000/?DEBUG_MODE&position=-100,100
+    http://localhost:3000/?DEBUG_MODE&position=-1,27
```

Comment:
Is this change necessary?

Reply:
I wanted to change these coords because the `-100,100` was giving me an error (probably because this test scene is broken) and I think it is more proper to give the users a valid example.

Reply:
I'm curious about the broken scene. Does it throw an exception? Does it load at all?

Reply:
The problem seems to be specifically in the test scene `-102,102`. I attach the errors throwed in the browser console:

![image](https://user-images.githubusercontent.com/64659061/128319141-692f156a-eff5-4317-ade0-2fddceb00750.png)
![image](https://user-images.githubusercontent.com/64659061/128319246-86a6a761-554e-4464-b41d-4c2e85a545e4.png)

Reply:
oh ok, that is not good. It seems like scenes can break the renderer. Could you track it as critical? People could not go to nearby locations CC @olavra

Reply:
Ok, we're going to create a task for this 👍